lib/recurly/subscription.php

```diff
@@ -136,4 +143,9 @@ protected function getWriteableAttributes() {
       'shipping_address', 'shipping_address_id'
     );
   }
+  private function isEmbedded($node) {
```

Comment:
I think we'd talked about moving this up the inheritance tree at one point since there's a few classes using it. What do you think about doing that as part of this PR?